lib/Myriad/Bootstrap.pm

```diff
@@ -171,50 +232,41 @@ sub boot {
                 die "excessive module loading detected: $loaded_modules" if $loaded_modules;
             }
 
-            my $stop = 0;
             local $SIG{HUP} = sub {
                 say "$$ - HUP detected in parent";
                 kill 3, $pid;
```

Comment:
You use names in `%SIG`. Why don't you use names here as well?

```
> perl -e 'kill QUIT => $$'
Quit
> echo $(($? & 0x7f))
3
```

As you can see, the process running that perl snippet committed suicide using signal 3 (aka `SIGQUIT`)